source/MaterialXGenGlsl/GlslResourceBindingContext.cpp

```diff
@@ -0,0 +1,45 @@
+//
+// TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXGenGlsl/GlslResourceBindingContext.h>
+
+namespace MaterialX
+{
+
+//
+// GlslResourceBindingContext
+//
+GlslResourceBindingContext::GlslResourceBindingContext()
+{
+    _requiredExtensions.insert("GL_ARB_shading_language_420pack");
```

Comment:
General question from before, do we need to at some point consider more than just ubo for 4.2? And if so would we create another binding context or try and handle it here ?

Reply:
If we intend to maintain glsl 4.0 baseline, UBO might be enough for now. For e.g. the sheen albedo table could become texture buffer object. But at this point we might be looking at a formal 4.5 code gen

Reply:
The same bindingcontext could be extended.